traefik/docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: '3.9'
+services:
+  traefik:
+    image: traefik:v2.9
+    container_name: srv_traefik
+    command:
+      ## Entrypoints
+      - --entryPoints.http.address=:80
+      - --entryPoints.https.address=:443
+      ## Logging
+      - --log=true
+      - --log.level=INFO
+      - --accesslog=true
+      ## API
+      - --api=true
+      - --api.dashboard=true
+      - --api.insecure=true
+      ## Providers
+      - --providers.docker=true
+      - --providers.docker.endpoint=unix:///var/run/docker.sock
+      - --providers.docker.network=reverse-proxy
+      - --providers.docker.exposedByDefault=false
+      ## Metrics
+      - --metrics.prometheus=true
+      - --metrics.prometheus.buckets=0.1,0.3,1.2,5.0
+      # Certificates
+      - --certificatesResolvers.le.acme.httpchallenge=true
+      - --certificatesResolvers.le.acme.httpchallenge.entrypoint=http
+      - --certificatesResolvers.le.acme.email=pedro.parqueador@galacticcoders.xyz
```

Comment:
maybe make this an environment variable?

Reply:
Sounds good.
Done